File: data/gpt-4o/readable_format_HPEC/cpp/file069.cpp
#include <stdio.h>
#include <omp.h>

int main(){
    int len = 100;
    int tmp = 10;
    int a[len];

#pragma omp parallel for
    for(int i = 0; i < len; i++){
        a[i] = tmp;
        tmp = a[i] + i;
    }

    printf("a[50] = %d
", a[49]);

    return 0;
}

File: data/gpt-4o/readable_format_HPEC/fortran/file069.f90
program DRB035_truedepscalar_orig_yes
    use omp_lib
    implicit none

    integer i, tmp, len
    integer, dimension(:), allocatable :: a

    len = 100
    tmp = 10
    allocate (a(len))

    !$omp parallel do
    do i = 1, len
        a(i) = tmp
        tmp = a(i) + i
    end do
    !$omp end parallel do

    print 100, a(50)
    100 format ('a(50) =',i3)

    deallocate(a)
end program
